test/access/HashRegistry.sol.ts

```diff
@@ -11,11 +11,12 @@ export async function signHash(
   timestamp: number
 ): Promise<BytesLike[]> {
   return Promise.all(
-    signers.map(async (signer) =>
-      signer.signMessage(
+    signers.map(async (signer) => {
+      const signature = await signer.signMessage(
```

Comment:
My race condition issues that cause tests to fail started when I removed these awaits, but oddly enough, putting them all back did not solve the issue completely

Reply:
Yeah good point, I'll try that